src/vue/common/fees/FeesViewer.vue

```diff
@@ -0,0 +1,258 @@
+            {{ 'fees-renderer.destination-fixed-fee-lbl' | globalize }}
+          </td>
+          <td>
+            <template v-if="paidForDestination">
+              {{ formatFee(0) }}
+            </template>
+            <template v-else>
+              {{ formatFee(fees.destination.fixed) }}
+            </template>
+          </td>
+        </tr>
+        <tr v-if="fees.destination.calculatedPercent">
+          <td>
+            {{ 'fees-renderer.destination-percent-fee-lbl' | globalize }}
+          </td>
+          <td>
+            <template v-if="paidForDestination">
+              {{ formatFee(0) }}
+            </template>
+            <template v-else>
+              {{
+                formatFee(
+                  fees.destination.calculatedPercent
+                )
+              }}
+            </template>
+          </td>
+        </tr>
+      </template>
+
+      <tr class="fees-renderer__total-fee-row">
+        <td>
+          {{ 'fees-renderer.source-fixed-total-fee-lbl' | globalize }}
+        </td>
+        <td>
+          <template v-if="paidForDestination">
+            {{
+              formatFeeSum([
+                fees.destination.fixed,
+                fees.source.fixed,
+                fees.destination.calculatedPercent,
+                fees.source.calculatedPercent,
+              ])
+            }}
+          </template>
+          <template v-else>
+            {{
+              formatFeeSum([
+                fees.source.fixed,
+                fees.source.calculatedPercent
+              ])
+            }}
+          </template>
+        </td>
+      </tr>
+
+      <tr
+        class="fees-renderer__total-fee-row"
+      >
+        <td>
+          {{ 'fees-renderer.destination-fixed-total-fee-lbl' | globalize }}
+        </td>
+        <td>
+          <template v-if="paidForDestination">
+            {{
+              formatFeeSum([0])
+            }}
+          </template>
+          <template v-else>
+            {{
+              formatFeeSum([
+                fees.destination.fixed,
+                fees.destination.calculatedPercent
+              ])
+            }}
+          </template>
+        </td>
+      </tr>
+
+      <tr
+        v-if="fees.isAnyDestinationFee && !fees.isWithdrawalFees"
+      >
+        <td class="fees-renderer__tick-field">
+          <tick-field v-model="isPaidForRecipient">
+            {{ 'fees-renderer.pay-fees-for-recipient-lbl' | globalize }}
+          </tick-field>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</template>
+<script>
+import IdentityGetterMixin from '@/vue/mixins/identity-getter'
+import FormMixin from '@/vue/mixins/form.mixin'
+import { CoupledFeesRecord } from './fees.record'
+
+const EVENTS = {
+  paidForDestination: 'paidForDestination',
+}
+
+export default {
+  name: 'fees',
+  mixins: [
+    FormMixin,
+    IdentityGetterMixin,
+  ],
+  props: {
+    fees: {
+      type: Object,
+      default: () => CoupledFeesRecord,
```

Comment:
```js
fees: {
  type: CoupledFeesRecord,
  required: true
}
```

fees prop is required cuz the component cannot go on without it. also it strongly relies on CoupledFeesRecord, so we should ensure the prop can accept instances of this class only.  Vue does the half of job for us